### PR TITLE
PAE-157 - Styling the header of the Regis theme.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR changes the Regis theme header to comply with the Regis mockups.

## Screenshots:

image
image
image

## Testing:

- Check out this branch.
- Enable Comprehensive theming: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html
- Compile assets, `paver update_assets --themes=pearson-regis-theme`
- Assign 'pearson-regis-theme' to a new site.

## Reviewers:

- [ ] @alexjmpb 
- [ ] @anfbermudezme

### CODE REVIEW

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss [hunk 1]
```diff
@@ -5,78 +5,86 @@
 // is imported after lms-main-v1.css on the courseware page and overwrites the theme rules.
 #top {
   &.global-header {
-    border-bottom: none;
+    border-bottom: 1px solid $gray;
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss]:
```suggestion
    border-bottom: 1px solid #D4D7D9;
    box-shadow: none;
    min-height: 100px;
    padding: 0 55px;

    .main-header {
      display: flex;
      align-items: center;
      gap: 60px;
      height: 100px;

      @include media-breakpoint-down(lg) {
        height: 100px;
        justify-content: space-between;
        
        h1 {
          a {
            margin: 0;
          }
        }
      }
    }

```

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss [hunk 2]
```diff
@@ -5,78 +5,86 @@
...
 
     .nav-links {
       .main {
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss:42]:
```suggestion
      .main {
        display: flex;
        gap: 30px;
        margin: 0;
        height: 100%;

```

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss [hunk 3]
```diff
@@ -5,78 +5,86 @@
 // is imported after lms-main-v1.css on the courseware page and overwrites the theme rules.
 #top {
   &.global-header {
-    border-bottom: none;
+    border-bottom: 1px solid $gray;
 
     .nav-links {
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss:35]:
```suggestion
    .nav-links {
      display: none;
      align-items: center;
      height: 100%;
      width: 100%;
      justify-content: space-between;

```

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss [hunk 4]
```diff
@@ -5,78 +5,86 @@
...
       .main {
         .nav-tab {
+
           a, .tab-nav-link {
-            color: $primary;
+            color: $gray;
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss:50]:
```suggestion
            color: $gray;
            display: flex;
            align-items: center;
            height: 100%;
            padding: 0;
            font-weight: 500;
            border-bottom: 5px solid transparent;

```

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss [hunk 5]
```diff
@@ -5,78 +5,86 @@
...
 
             &.active,
             &:hover {
-              border-bottom-color: $secondary;
+              border-bottom-color: $primary;
               font-weight: 700;
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss]:
```suggestion
              border-bottom-color: $primary-light;
```

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss [hunk 6]
```diff
@@ -5,78 +5,86 @@
...
             }
           }
         }
       }
 
       .secondary {
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss:66]:
```suggestion
      .secondary {
        display: flex;
        align-items: center;
        margin: 0;
        gap: 30px;

        .nav-item {
          margin: 0;
          
          a {
            font-weight: 500;
          }
        }
```

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss [hunk 7]
```diff
@@ -5,78 +5,86 @@
...
-        color: $primary;
+        .user-image-frame {
+          border: none;
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss:83]:
```suggestion
          border: none;
          height: 30px;
```

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss [hunk 8]
```diff
@@ -5,78 +5,86 @@
...
+        }
+      }
 
-        .toggle-user-dropdown {
-          i {
-            padding: 0 0.5em;
+      @include media-breakpoint-up(lg) {
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss:87]:
```suggestion
      @include media-breakpoint-up(lg) {
        display: flex;
```

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss [hunk 9]
```diff
@@ -5,78 +5,86 @@
...
           }
         }
+      }
+    }
 
-        .nav-item {
-          a, .help-link {
-            color: $primary;
+    .header-logo {
+      a {
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss]:
```suggestion
    .header-logo {
      margin: 0;

      a {
        margin: 0;

        .logo {
          height: 30px;
          margin: 0;
        }

```

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss [hunk 10]
```diff
@@ -5,78 +5,86 @@
...
-    &.global-header {
+    // Responsive styling for mobile
+    @include media-breakpoint-down(md) {
       .hamburger-menu {
-        top: 27px;
-        left: unset;
-        right: 30px;
-        width: 18px;
+        left: initial;
+        @include right($baseline);
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss]:
```suggestion
        @include right($baseline);
        position: relative;
        top: 0;
        left: 0;
        padding: 0;
```

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss [hunk 11]
```diff
@@ -5,78 +5,86 @@
 // is imported after lms-main-v1.css on the courseware page and overwrites the theme rules.
 #top {
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_navigation.scss:6]:
```suggestion
#top {
  @include media-breakpoint-down(lg) {
    padding: 0 15px !important;
  }

```

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss [hunk 12]
```diff
@@ -7,7 +7,7 @@ $secondary: #E89D00;
 $footer-background: #242E33;
 $footer-contrast: #F2F2F2;
 $dark: #313131;
-$gray: #E6E6E6;
+$gray: #242E33;
```
Comment (alexjmpb) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss:13]:
```suggestion
$primary-light: #EE3F40;

```

**Review by alexjmpb — CHANGES_REQUESTED** [review]
All comments are suggestions to improve fidelity to FIgma doc. Would you consider to apply them?

When applied the result should be the following:
Screenshot 2022-07-15 at 23-29-10 Dashboard Your Platform Name Here
Screenshot 2022-07-15 at 23-29-55 Dashboard Your Platform Name Here
Screenshot 2022-07-15 at 23-30-15 Signed Out Your Platform Name Here

**Comment by Squirrel18** [comment]
Really great suggestions, good work, thanks @alexjmpb

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss (hunk 12 above)
Comment (alexjmpb) [reply edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss:13]: Would you consider add the variable $primary-light: #EE3F40; and use it for the color border on line 59 of _navigation.scss

**Review by alexjmpb — CHANGES_REQUESTED** [review]

**Comment by alexjmpb** [comment]
> Really great suggestions, good work, thanks @alexjmpb

Thanks, no worries!!

**Review by alexjmpb — APPROVED** [review]

**Pull request merged**